contrib/metallb/roles/provision/defaults/main.yml

```diff
@@ -13,4 +13,4 @@ metallb:
     cpu: "100m"
     memory: "100Mi"
   port: "7472"
-  version: v0.7.3
+  version: v0.9
```

Comment:
We might want to pin to v0.9.0 if such a tag is available.

Reply:
Thanks for your review @Miouge1 , a nice point.
The latest stable tag is v0.9.3 and I tested it actually as https://hub.docker.com/layers/metallb/controller/v0.9.3/images/sha256-c8b0da00dd83db99bf00fb7088c33e7aaf52fa679f962610f1fe5ed173f66b77?context=explore
I will update here for v0.9.3.